README.md

```diff
@@ -1 +1,15 @@
-# Neptune
+# Neptune Documentation
+
+## ReadTheDocs
+
+The documentation in this repository is automatically rendered at [Read the
+Docs](https://excalibur-neptune.readthedocs.io/en/latest/).
+
+Each branch and pull request is also rendered. To view these, go to the [Read
+the Docs](https://excalibur-neptune.readthedocs.io/en/latest/) page, and then
+click on "Read the Docs" at the bottom of the sidebar.
+Click "Builds" (under "On Read the Docs") and select the relevant branch or pull request from the list.
+Click on the small grey "View Docs" link to the right of the "Build Completed"
+badge.
```

Comment:
I've set up the webhook correctly now, so it should be viewable by going to the PR and clicking the readthedocs status check link

Reply:
Ah thanks!  Actually, are branches currently set up to build? There's `add-docs`, but I don't see any of the other branches (though I made this PR shortly after pushing, so perhaps that prevents it building the branch separately?)